README.md

```diff
@@ -776,6 +780,167 @@ You'll see 504 gateway timeout 10 sec after you send the request
 ```
 See also [Request Timeouts](https://istio.io/latest/docs/tasks/traffic-management/request-timeouts/)
 
+#### Connect Timeout
+The configuration below set 4 sec tcp connection timeout for a request to the non-existent service.
+
+Prepare a non-existent service and check the connect timeout.
+The ip in the Endpoints resource should be a non-existent IP address.
+
+```yaml
+# kubectl apply -f manifests/non-existent-service.yaml -n testns1
+
+kubectl apply -n testns1 -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: non-existent-service
+spec:
+  clusterIP: None
+---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: non-existent-service
+subsets:
+  - addresses:
+    - ip: <non-existent IP address>
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: non-existent-service
+spec:
+  hosts:
+  - non-existent-service.testns1.svc.cluster.local
+  ports:
+  - number: 80
+    name: http
+    protocol: HTTP
+  resolution: DNS
+  location: MESH_EXTERNAL
+EOF
+```
+
+Set the tcp connection timeout in spec.trafficPolicy.connectionPool.tcp.connectTimeout in DestinationRule.
+(The default is 10 sec.)
+```yaml
+# kubectl apply -f manifests/virtualservice-non-existent-service.yaml -n testns1
+
+kubectl apply -n testns1 -f - <<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: non-existent-service
+spec:
+  hosts:
+  - non-existent-service.testns1.svc.cluster.local 
+  http:
+  - route:
+    - destination:
+        host: non-existent-service.testns1.svc.cluster.local 
+        port:
+          number: 80
+    retries:
+      attempts: 0
+EOF
+
+# kubectl apply -f manifests/destination-rule-connect-timeout.yaml -n testns1
+
+kubectl apply -n testns1 -f - <<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: non-existent-service
+spec:
+  host: non-existent-service.testns1.svc.cluster.local
+  trafficPolicy:
+    connectionPool:
+      tcp:
+        connectTimeout: 4s
+EOF
+```
+
+Send the HTTP request to http://non-existent-srevice.testns1.svc.cluster.local/. The IP address of this endpoint is invalid, so the request will fail and trigger a connection timeout after 4 sec.
```

Comment:
Sorry, my suggestion has typo. Please fix it.

```suggestion
Send the HTTP request to http://non-existent-service.testns1.svc.cluster.local/. The IP address of this endpoint is invalid, so the request will fail and trigger a connection timeout after 4 sec.
```

Reply:
I'm sorry I didn't notice.
Committed it.